rest/adminapitest/admin_api_test.go

```diff
@@ -3776,6 +3776,63 @@ func TestSetFunctionsWhileDbOffline(t *testing.T) {
 	resp.RequireResponse(http.StatusOK, syncFunc)
 }
 
+func TestCollectionSyncFnWithBackticks(t *testing.T) {
+	base.SetUpTestLogging(t, base.LevelInfo, base.KeyHTTP)
+
+	// Start SG with bootstrap credentials filled
+	ctx := base.TestCtx(t)
+	config := rest.BootstrapStartupConfigForTest(t)
+	sc, err := rest.SetupServerContext(ctx, &config, true)
+	require.NoError(t, err)
+	serverErr := make(chan error, 0)
+	go func() {
+		serverErr <- rest.StartServer(ctx, &config, sc)
+	}()
+	require.NoError(t, sc.WaitForRESTAPIs())
+	defer func() {
+		sc.Close(ctx)
+		require.NoError(t, <-serverErr)
+	}()
+
+	// Get a test bucket, and use it to create the database.
+	tb := base.GetTestBucket(t)
+	defer tb.Close()
+
+	rt := rest.NewRestTester(t, &rest.RestTesterConfig{PersistentConfig: true})
+	defer rt.Close()
+
+	scopesConfig := rest.GetCollectionsConfig(t, tb, 1)
+	dataStoreNames := rest.GetDataStoreNamesFromScopesConfig(scopesConfig)
+	scopeName, collectionName := dataStoreNames[0].ScopeName(), dataStoreNames[0].CollectionName()
+	// Initial DB config
+	dbConfig := fmt.Sprintf(`{
+    "num_index_replicas": 0,
+    "bucket": "%s",
+    "scopes": {
+      "%s": {
+         "collections": {
+            "%s": {
+               "sync":`, tb.GetName(), scopeName, collectionName) + "`" +
+		`function(doc, oldDoc, meta) {
+					var owner = doc._deleted ? oldDoc.owner : doc.owner;
+					requireUser(owner);
+					var listChannel = 'lists.' + doc._id;
+					var contributorRole = 'role:' + listChannel + '.contributor';
+					role(owner, contributorRole);
+					access(contributorRole, listChannel);
+					channel(listChannel);
+				}` + "`" + `
+         	}
+}
+      	}
+	  }
+ 	}`
+
+	// Create initial database
+	resp := rt.SendAdminRequest(http.MethodPut, "/db/", dbConfig)
+	assert.Equal(t, resp.Code, http.StatusCreated)
```

Comment:
let's also make sure that we can update the config too here, with the same config, PUT /db/_config